### PR TITLE
Adds Banksy Farm DAO strategy [banksy-dao]

### PR DESCRIPTION
**Pull request by BanksyFarm**
Changes proposed in this pull request:
- Adds support for reading a value in NFT and adding it. (Same as Sandman Finance)

### CODE REVIEW

File: src/strategies/banksy-dao/index.ts [hunk 1]
```diff
@@ -0,0 +1,95 @@
+import { Multicaller } from '../../utils';
+import { BigNumber } from '@ethersproject/bignumber';
+
+export const author = 'myst4';
```
Comment (ChaituVR) [inline src/strategies/banksy-dao/index.ts:4]:
```suggestion
export const author = 'BanksyFarm';
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Hi @jordanmessina Looks like this is same as https://github.com/snapshot-labs/snapshot-strategies/pull/324/files why not reuse it?

**Comment by sandmanfinance** [comment]
Although very similar, the strategies are for two different NFT contracts.
If you check, the ABIs are different.
So they must be different strategies.

Thank you very much.

File: src/strategies/banksy-dao/index.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  }
+  const walletIDToSkills: Record<
+    string,
+    any
+  > = await callTokenToSkill.execute();
+  
+  const results = {} as Record<string, number>;
+  for (const [walletID, values] of Object.entries(walletIDToSkills)) {
+    const address = walletID.split('-')[0];
+    console.log(values[1]);
```
Comment (ChaituVR) [inline src/strategies/banksy-dao/index.ts:78]:
```suggestion
```

**Review by ChaituVR — CHANGES_REQUESTED** [review]

**Comment by ChaituVR** [comment]
Yea , looks very similar except `getArtWorkOverView`

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**